messages/en.json

```diff
@@ -31,7 +31,9 @@
         "PING! {nick}",
         "I'm sorry Dave, I'm afraid I can't do that.",
         "Give me a ping, Vasily. One ping only, please.",
-        "Would you people please leave me alone? Seriously."
+        "Would you people please leave me alone? Seriously.",
+        "Did you know the plural of lykos is lykoi?",
```

Comment:
I thought about that, but figured that it's still a valid trivia piece because the project is called lykos. But I can also remove that one.